src/Library/types.ts

```diff
@@ -0,0 +1,85 @@
+export type Author = {
+    readonly name: string;
+    readonly url: string;
+    readonly position: string;
+};
+
+enum DefaultCategories {
+    All = 'All',
+    Popular = 'Popular'
+}
+
+enum PartnersCategories {
+    Reastream = 'Reastream'
+}
+
+export type LibraryCategories = DefaultCategories | PartnersCategories;
+
+export enum LibraryDirections {
+    All = 'All',
+    Graphics = 'Graphics',
+    Dev = 'Dev',
+    Marketing = 'Marketing',
+    Management = 'Management',
+    Advertising = 'Advertising'
+}
+
+export enum LibraryFormats {
+    All = 'All',
+    Conference = 'Conference',
+    Video = 'Video',
+    Tutorial = 'Tutorial',
+    Interview = 'Interview'
+}
+
+export enum LibraryLevels {
+    All = 'All',
+    Beginner = 'Beginner',
+    Specialist = 'Specialist'
+}
+
+enum VideoSize {
+    Default = 'Default',
+    Featured = 'Featured'
+}
+
+export interface Video {
+    readonly id: number;
+    readonly title: string;
+    readonly description: string;
+    readonly mainCategory?: LibraryCategories;
+    readonly categories: LibraryCategories[];
+    readonly publishDate?: string;
+    readonly authors: Author[];
+    readonly preview: string;
```

Comment:
What the `preview` describes? It might be it describes if we should show the preview of a vodeo? In that case it should be `Boolean`.
It might be I didn't get it and I'm wrong, just want to pay your attention to it.

Reply:
It is just a source image preview.